src/simulator_core/entities/assets/demand_cluster.py

```diff
@@ -29,50 +54,159 @@ def __init__(
             self,
             asset_name: str,
             asset_id: str,
-            panda_pipes_net: pandapipesNet
+            pandapipe_net: pandapipesNet
     ):
-        """Initialize a ProductionCluster object.
+        """Initialize a DemandCluster object.
 
         :param str asset_name: The name of the asset.
         :param str asset_id: The unique identifier of the asset.
         :param PandapipesNet pandapipe_net: Pandapipes network object to register asset to.
         """
-        super().__init__(asset_name, asset_id, panda_pipes_net)
+        super().__init__(asset_name=asset_name, asset_id=asset_id, pandapipe_net=pandapipe_net)
+
+        self.setpoints = dict()
+        self.parameters = dict()
+        self.states = dict()
+
+        self.internal_diameter = DEFAULT_DIAMETER
+        self.temperature_supply = DEFAULT_TEMPERATURE
+        self.temperature_return = DEFAULT_TEMPERATURE - DEFAULT_TEMPERATURE_DIFFERENCE
+        self.temperature_return_target = self.temperature_return
+        self.pressure_input = DEFAULT_PRESSURE
+        self.thermal_power_allocation = 0
+        self.mass_flowrate = 0
+
+        # Objects of the asset
+        self._initialized = False
+        self._intermediate_junction: Optional[Junction] = None
+        self._flow_control = None
+        self._heat_exchanger = None
+        self._simulated = False
+        self._from_junction = None
+        self._to_junction = None
```

Comment:
Why do you define a self._from_junction? This can be confusing with the self.from_junction already defined in the asset abstract class?